### DISCUSSION TITLE
tabset panel border-color rendering

### DISCUSSION THREAD

**Discussion by KaiWaldrant** [quarto-dev/quarto-cli#4861] [Feature Requests]
I was making a bug issue about the tabset-panel inconsistency in border-colors used when I found out it was actually an issue with the themes or rather as i think about it a combination of quarto and the themes used. You can find a demo of the issue here: https://github.com/KaiWaldrant/quarto-panel-tabset-border

When using `.tabset-panel` quarto adds a `border-color` for the div with a class `tab-content`. When using a theme like 'simplex' for light-mode and 'darkly' for dark-mode. This will come in conflict as both of those themes do not have a border-color for the `tab-content` class.

I am unsure how to classify this issue and where to post it

**Comment by dragonstyle** [comment]
This is a great find - I've opened an issue to track fixing the problem. There are really two issues:

1) Quarto is incorrectly using the scss `$border-color` for the tab body instead of using the tab specific border color. Often, these are identical or very similar, but sometimes they are not!

2) Simplex provides a custom border color for nav tabs, but doesn't provide a custom border color the active tab. This means that the active tab is getting the color of the base bootstrap theme, which is just slightly different than theirs. This is something Quarto could fix by providing a theme specific override for the value.

I'm testing a fix for 1 now and will take a quick peak to see if fixing 2 is straightforward...

Thanks for reporting!

**Comment by mcanouil** [comment]
Closing as resolved and in favour of https://github.com/quarto-dev/quarto-cli/issues/4868
